Formula/stubby.rb

```diff
@@ -0,0 +1,55 @@
+class Stubby < Formula
+  desc "DNS privacy enabled stub resolver service based on getdns"
+  homepage "https://getdnsapi.net/blog/dns-privacy-daemon-stubby/"
+  url "https://github.com/getdnsapi/stubby/archive/v0.1.0.tar.gz"
+  sha256 "6210291850d6f7f124a5ba4bcee2f50814f020b7a0c4e67c6646bfe35ed5dd5b"
+  head "https://github.com/getdnsapi/stubby.git", :branch => "develop"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "getdns" => :build
+  depends_on "libtool" => :build
+
+  def install
+    system "autoreconf", "-fiv"
+    system "./configure", "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}",
+                          "--sysconfdir=#{etc}"
+    system "make", "install"
+  end
+
+  plist_options :startup => true, :manual => "sudo stubby -C #{HOMEBREW_PREFIX}/etc/stubby/stubby.conf"
+
+  def plist; <<-EOS.undent
+    <?xml version="1.0" encoding="UTF-8"?>
+    <!DOCTYPE plist PUBLIC "-/Apple/DTD PLIST 1.0/EN" "http:/www.apple.com/DTDs/PropertyList-1.0.dtd">
+    <plist version="1.0">
+      <dict>
+        <key>Label</key>
+        <string>#{plist_name}</string>
+        <key>KeepAlive</key>
+        <true/>
+        <key>RunAtLoad</key>
+        <true/>
+        <key>ProgramArguments</key>
+        <array>
+          <string>#{opt_bin}/stubby</string>
+          <string>-C</string>
+          <string>#{etc}/stubby/stubby.conf</string>
+        </array>
+        <key>StandardErrorPath</key>
+        <string>#{var}/log/stubby/stubby.log</string>
+        <key>StandardOutPath</key>
+        <string>#{var}/log/stubby/stubby.log</string>
+      </dict>
+    </plist>
+    EOS
+  end
+
+  test do
+    (testpath/"stubby_test.conf").write("{ listen_addresses: [ 127.0.0.1@5553 ] }")
+    output = shell_output("#{bin}/stubby -i -C stubby_test.conf 2>/dev/null")
```

Comment:
I tried creating a test that started stubby as a daemon but it seems the test framework is limited and that didn't work... (test function attached). If you can guide me on how to do this correctly would be appreciated. Otherwise this will require development work just to satisfy this request...... However one caveat is that any such test depends on connectivity to one of a set of experimental test servers which have no SLA so it could fail simply because the servers are down or several other reasons so it isn't wholly reliable anyway. 

In the meantime I have created a Tap which several people have used with no issues. It would be nice to have #15578 accepted regardless since it is the correct behaviour for getdns 1.1.2.